### DISCUSSION TITLE
Repo doesn't shows any language

### DISCUSSION THREAD

**Discussion by FabianWassermann** [github-linguist/linguist#5411] [Classification] [answered]
Hey. Why there is not language analytic? https://github.com/FabianWassermann/phpAutoCompletion

**Comment by lildude** [comment, marked as answer]
Because all of the files in your repo are either considered prose (the markdown files) or data (the json and dot files) and thus not counted and your one "code" file has an extension `.code-snippets` that Linguist doesn't know about.

If you want that file to count towards your language you'll either need to change the extension, or implement a language override.

**Reply by FabianWassermann** [reply]
Thanks for info.

**Comment by BOSOEK** [comment]
I also deleted all the files and re-uploaded, but the repository does not recognize any language.   https://github.com/BOSOEK/MS_Pytorch

**Reply by lildude** [reply]
@BOSOEK wrong discussion 😉 You should really start your own discussion.
